website/src/models/task.js

```diff
@@ -50,10 +50,14 @@ var checklist = [{
 
 var DailySchema = new Schema(
   _.defaults({
-    type: {type:String, 'default': 'daily'},
+    type: {type: String, 'default': 'daily'},
+    //TODO: Cleaner to store interval as enum instead of str?
+    frequency: {type: String, 'default': 'daily'}, // 'daily', 'weekly'
```

Comment:
@ your first comment, what's the best way & place for me to declare this as an enum? Doesn't seem like js has traditional enums, and I'm not too familiar with conventions for js nor for habitrpg yet.

@ second comment, good call! I'm starting to understand that, in mongoose world, the properties of schemas are basically part of the public API, at least the way we use them here. I'll make the change.